Cargo.toml

```diff
@@ -18,45 +18,45 @@ all-features = true
 # These two are generally really common simple dependencies so it does not seem
 # much of a point to optimize these, but these could in theory be removed for
 # an indirection through std::Formatter.
-dtoa = "0.4"
```

Comment:
Bumping the patch version only restricts downstream users to that patch version or up. Unless we know that the version is in fact a minimum we need shouldn't increase it but instead let users do `cargo update` if they want to update.